met-web/src/components/engagement/new/view/SuggestedEngagements.tsx

```diff
@@ -1,143 +1,89 @@
-    }, [engagement]);
-
-    const pages = Math.ceil((totalEngagements - 1) / pageLength);
-    const engagementSlice = suggestedEngagements.slice(page * pageLength, (page + 1) * pageLength);
+    const suggestedEngagementsPromise = getEngagements({ size: 5, page: 1, include_banner_url: true }).then(
+        async (response) => {
+            const currentEngagement = await engagement;
+            return response.items
+                .filter((engagement) => engagement.id !== currentEngagement.id)
+                .slice(0, 4)
+                .map((value) => ({ value, sort: Math.random() }))
+                .sort((a, b) => a.sort - b.sort)
+                .map(({ value }) => value);
+        },
+    );
 
     return (
         <section id="suggested-engagements" aria-labelledby="suggested-engagements-header">
-            <Box sx={{ padding: { xs: '43px 16px 24px 16px', md: '32px 5vw 40px 5vw', lg: '32px 156px 40px 156px' } }}>
-                <Header2 weight="thin" decorated id="suggested-engagements-header">
+            <Box sx={{ padding: { xs: '64px 16px 24px 16px', md: '64px 5vw 40px 5vw', lg: '64px 156px 40px 156px' } }}>
+                <Header2 weight="thin" sx={{ mb: '10px' }} decorated id="suggested-engagements-header">
                     You may also be interested in
                 </Header2>
-                <Grid container justifyContent="center" alignItems={'center'} direction={'row'}>
-                    <Button variant="tertiary" disabled={page < 1} onClick={() => setPage(Math.max(page - 1, 0))}>
-                        <FontAwesomeIcon icon={faChevronLeft} />
-                    </Button>
-                    {isLoading
-                        ? Array.from({ length: 3 }).map((_, index) => (
-                              <RepeatedGrid
-                                  times={pageLength - 1}
-                                  item
-                                  width="320px"
-                                  margin="0 16px"
-                                  sx={{
-                                      flexBasis: '320px',
-                                      alignItems: 'center',
-                                      justifyContent: 'center',
-                                  }}
-                              >
-                                  <TileSkeleton />
-                              </RepeatedGrid>
-                          ))
-                        : engagementSlice.map((engagement, index) => (
-                              <Grid item width="320px" margin="0 16px" key={engagement.id}>
-                                  <EngagementTile passedEngagement={engagement} engagementId={engagement.id} />
-                              </Grid>
-                          ))}
-                    <Button
-                        variant="tertiary"
-                        disabled={page > pages - 2}
-                        onClick={() => setPage(Math.min(page + 1, pages - 1))}
+                <Grid
```

Comment:
With search results or a sequence of related items, a list tag can help orient the user - helping them understand that they're browsing a collection.

I don't think engagement cards should ever appear on their own right? We could consider a `<ul><li><Card /></li></ul>` pattern here. It's what I've done for the screen reader ticket on the search results page. Curious to hear your thoughts